Packs/DSPM/Integrations/DSPM/DSPM.yml

```
@@ -0,0 +1,984 @@
+    name: dspm-update-alert-status
+    outputs:
+      - contextPath: DSPM.AlertStatusUpdate.newStatus
+        description: 'Updated alert status.'
+        type: String
+      - contextPath: DSPM.AlertStatusUpdate.oldStatus
+        description: 'Old alert status.'
+        type: String
+      - contextPath: DSPM.AlertStatusUpdate.alertId
+        description: 'Alert id.'
+        type: String
+  - arguments:
+    - description: 'Exact detection time (equals).'
+      isArray: false
+      name: detection_time_equals
+      required: false
+      secret: false
+    - description: 'Detection time (greater than or equal).'
+      isArray: false
+      name: detection_time_greater_than_or_equal
+      required: false
+      secret: false
+    - description: 'Detection time (greater than).'
+      isArray: false
+      name: detection_time_greater_than
+      required: false
+      secret: false
+    - description: 'Detection time (less than or equal).'
+      isArray: false
+      name: detection_time_less_than_or_equal
+      required: false
+      secret: false
+    - description: 'Detection time (less than).'
+      isArray: false
+      name: detection_time_less_than
+      required: false
+      secret: false
+    - description: 'List of policy names.'
+      isArray: true
+      name: policy_name_in
+      required: false
+      secret: false
+    - description: 'Exact policy name.'
+      isArray: false
+      name: policy_name_equals
+      required: false
+      secret: false
+    - description: 'List of asset names.'
+      isArray: true
+      name: asset_name_in
+      required: false
+      secret: false
+    - description: 'Exact asset name.'
+      isArray: false
+      name: asset_name_equals
+      required: false
+      secret: false
+    - description: 'List of cloud providers ["AWS", "AZURE", "GCP", "SNOWFLAKE", "FILE_SHARE", "O365"].'
+      isArray: true
+      name: cloud_provider_in
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'AWS'
+      - 'AZURE'
+      - 'GCP'
+      - 'SNOWFLAKE'
+      - 'FILE_SHARE'
+      - 'O365'
+    - description: 'Exact cloud provider ["AWS", "AZURE", "GCP", "SNOWFLAKE", "FILE_SHARE", "O365"].'
+      isArray: false
+      name: cloud_provider_equals
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'AWS'
+      - 'AZURE'
+      - 'GCP'
+      - 'SNOWFLAKE'
+      - 'FILE_SHARE'
+      - 'O365'
+    - description: 'List of destination project vendor names.'
+      isArray: true
+      name: destination_project_vendor_name_in
+      required: false
+      secret: false
+    - description: 'Exact destination project vendor name.'
+      isArray: false
+      name: destination_project_vendor_name_equals
+      required: false
+      secret: false
+    - description: 'List of cloud environments ["UNKNOWN", "DEVELOPMENT", "STAGING", "TESTING", "PRODUCTION"].'
```

```suggestion
    - description: 'List of cloud environments.'
```
Is this a comma-separated list?  If so, it should be written as:
A comma-separated list of cloud environments.